databricks_cli/configure/config.py

```diff
@@ -28,54 +28,39 @@
 
 from databricks_cli.configure.provider import DEFAULT_SECTION, get_config_for_profile
 from databricks_cli.utils import error_and_quit
-from databricks_cli.sdk import ApiClient, DbfsService, WorkspaceService, JobsService, \
-    ClusterService, ManagedLibraryService
+from databricks_cli.sdk import ApiClient
 
 
 def require_config(function):
```

Comment:
maybe call it `@provide_api_client`. it is what the method does and the CLI functions shouldn't care if it is built from profile or some other ways. 

nit: Also what if `api_client` is already given? I think in this case it should simply pass it over. It might help testing.